src/types.ts

```diff
@@ -15,7 +15,6 @@ export interface EventObject {
    * The type of event that is sent.
    */
   type: string;
```

Comment:
```suggestion
  type: string;
  [key: string]: any;
```

Let's add this back for now. Too much code would be affected by this change, unfortunately.